src/integrations/feed/generators.ts

```diff
@@ -111,22 +110,15 @@ export const feedGenerators: Record<FeedVersion, FeedGenerator> = Object.freeze(
       }),
     ),
     popular: new FeedGenerator(
-      feedClient,
-      new SimpleFeedConfigGenerator({
-        providers: {
-          fresh: {
-            enable: true,
-            remove_engaged_posts: true,
-            page_size_fraction: 0.1,
-          },
-          engaged: {
-            enable: true,
-            remove_engaged_posts: true,
-            page_size_fraction: 1,
-            fallback_provider: 'fresh',
-          },
+      new FeedClient(process.env.POPULAR_FEED),
+      new FeedPreferencesConfigGenerator(
+        {},
+        {
+          includePostTypes: true,
+          includeBlockedSources: true,
```

Comment:
Does this mean we can more easily actually decide to block them now as well?
What was the reason we dont block them here?

Feel if you said you really hate a source and block it, it should not show anywhere basically 👀

Reply:
When it wasn't part of the feed service originally it was harder to do it.
Now with feed service everything is encapsulated and turned into config, so for api it's just turning on a flag :P
Pretty slick ah?

Reply:
100% but we want to keep exposing blocked stuff there?

Reply:
Oh amazing! Misread it 🤧
Pretty cool 🙏